src/router.js

```diff
@@ -0,0 +1,21 @@
+const { getRoot, getFromPublic, getAutocomplete } = require('./handlers');
+
+const router = (req, res) => {
+  const endPoint = req.url;
+  const { method } = req;
+
+  if (endPoint === '/') {
+    getRoot(res);
+  } else if (endPoint.includes('/autocomplete') && method === 'GET') {
+    getAutocomplete(req, res, endPoint);
+  } else if (endPoint.indexOf('/') !== -1) {
```

Comment:
this means that if I send a request to lets say `/elephant` then the condition will be met(since your condition will always be true)? it will call the function `getFromPublic` on it and it will return a server error(since there will be no file matching) while the true error should be 404 not found. 

So, it is better to have a condition for all the files you have in the public director(hint: add /public to all your static files href in html and then you can have this condition `endpoint.includes('public')`)

Reply:
Thank you for your help, you told us the problem and a hint to fix it 💟